docs/documentation/_data/deckhouse-alerts.yml

```
@@ -2951,27 +3323,34 @@ alerts:
       module: upmeter
       edition: ce
       description: |
-        Average probe pods count per upmeter-agent pod: {{ $value }}.
+        The average number of probe Pods per `upmeter-agent` Pod is {{ $value }}.
+
+        `upmeter-agent` is expected to clean Deployments created by the `control-plane/controller-manager` probe,
+        and `kube-controller-manager` should remove the associated Pods.
+        There should not be more of these Pods than there are master nodes (since `upmeter-agent` runs as a DaemonSet with a master `nodeSelector`), and the Pods should be deleted within seconds.
 
-        Upmeter agents should clean Deployments produced by control-plane/controller-manager probe,
-        and hence kube-controller-manager should clean their pods. There should not be more of these pods than
-        master nodes (upmeter-agent is a DaemonSet with master nodeSelector). Also, they should be
-        deleted within seconds.
+        This may indicate:
 
-        This might be an indication of a problem with kube-apiserver or kube-controller-manager. Or, probably,
-        the pods were left by old upmeter-agent pods due to Upmeter update.
+        - A problem with the `kube-apiserver` or `kube-controller-manager`.
+        - Stale Pods left from outdated `upmeter-agent` Pods after an Upmeter update.
 
-        1. Check upmeter-agent logs
+        Troubleshooting steps:
 
-        `kubectl -n d8-upmeter logs -l app=upmeter-agent --tail=-1 | jq -rR 'fromjson? | select(.group=="control-plane" and .probe == "controller-manager") | [.time, .level, .msg] | @tsv'`
+        1. Check the `upmeter-agent` logs:
+
+           ```shell
+           kubectl -n d8-upmeter logs -l app=upmeter-agent --tail=-1 | jq -rR 'fromjson? | select(.group=="control-plane" and .probe == "controller-manager") | [.time, .level, .msg] | @tsv'
+           ```
 
-        2. Check that control plane is functional, kube-controller-manager in particular.
+        2. Ensure the control plane is operating normally. Pay close attention to `kube-controller-manager`.
 
-        3. Delete pods manually:
+        3. Delete stale Pods manually:
 
-        `kubectl -n d8-upmeter delete po -l upmeter-probe=controller-manager`
+           ```shell
+           kubectl -n d8-upmeter delete po -l upmeter-probe=controller-manager
```

It is suggested to specify full commands everywhere, not aliases. 

```suggestion
           kubectl -n d8-upmeter delete pods -l upmeter-probe=controller-manager
```